File: khalti-android/src/main/java/com/khalti/checkOut/helper/MerchantPreferencePojo.java
package com.khalti.checkOut.helper;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class MerchantPreferencePojo {

    @SerializedName("sdk_position")
    @Expose
    private List<String> sdkPosition = new ArrayList<>();

    public List<String> getSdkPosition() {
        return sdkPosition;
    }
}

File: khalti-android/src/main/java/com/khalti/checkout/helper/MerchantPreferencePojo.kt
package com.khalti.checkout.helper


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class MerchantPreferencePojo {

    @SerializedName("sdk_position")
    @Expose
    val sdkPosition: List<String> = ArrayList()
}